src/main/java/core/basesyntax/enums/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.enums;
+
+public enum Color {
```

Comment:
The access modifier for the enum 'Color' is missing. It's a good practice to define it explicitly even if it's intended to be package-private.